src/test/java/edu/ucla/library/bucketeer/JobFactoryTest.java

```diff
@@ -56,4 +58,15 @@ public final void testCreateJob() throws ProcessingException, IOException {
         assertEquals(expected, job.setSlackHandle(SLACK_HANDLE).toJSON());
     }
 
+    /**
+     * Test JobFactory rejects files with duplicate headers.
+     */
+    @Test
```

Comment:
I think we need to throw a ProcessingException here and check that it's thrown. Returning a Job with no items doesn't tell the user why their CSV upload didn't work (since they don't see the logs). The code that calls `createJob` is already set up to catch this exception and set the HTTP response appropriately you just need to build the exception (and throw it) in addition to the logging you have in place.

Reply:
Modified test to verify ProcessingException is thrown and content of message